src/Core/hco_interp_mod.F90

```diff
@@ -1000,235 +854,12 @@ END SUBROUTINE ModelLev_Interpolate
-
-       ! Level 5:
-       Lct%Dct%Dta%V3(T)%Val(:,:, 5) = 0.139_sp   * REGR_4D(:,:,10,T) &
-                                     + 0.216_sp   * REGR_4D(:,:,11,T) &
-                                     + 0.216_sp   * REGR_4D(:,:,12,T) &
-                                     + 0.216_sp   * REGR_4D(:,:,13,T) &
-                                     + 0.214_sp   * REGR_4D(:,:,14,T)
-
-       ! Level 6:
-       Lct%Dct%Dta%V3(T)%Val(:,:, 6) = 2.20e-2_sp * REGR_4D(:,:,14,T) &
-                                     + 0.275_sp   * REGR_4D(:,:,15,T) &
-                                     + 0.275_sp   * REGR_4D(:,:,16,T) &
-                                     + 0.275_sp   * REGR_4D(:,:,17,T) &
-                                     + 0.173_sp   * REGR_4D(:,:,18,T)
-
-       ! Level 7:
-       Lct%Dct%Dta%V3(T)%Val(:,:, 7) = 0.130_sp   * REGR_4D(:,:,18,T) &
-                                     + 0.345_sp   * REGR_4D(:,:,19,T) &
-                                     + 0.345_sp   * REGR_4D(:,:,20,T) &
-                                     + 0.170_sp   * REGR_4D(:,:,21,T)
-
-       ! Level 8:
-       Lct%Dct%Dta%V3(T)%Val(:,:, 8) = 0.214_sp   * REGR_4D(:,:,21,T) &
-                                     + 0.416_sp   * REGR_4D(:,:,22,T) &
-                                     + 0.370_sp   * REGR_4D(:,:,23,T)
-
-       ! Level 9:
-       Lct%Dct%Dta%V3(T)%Val(:,:, 9) = 5.49e-2_sp * REGR_4D(:,:,23,T) &
-                                     + 0.490_sp   * REGR_4D(:,:,24,T) &
-                                     + 0.455_sp   * REGR_4D(:,:,25,T)
-
-       ! Level 10:
-       Lct%Dct%Dta%V3(T)%Val(:,:,10) = 4.06e-2_sp * REGR_4D(:,:,25,T) &
-                                     + 0.576_sp   * REGR_4D(:,:,26,T) &
-                                     + 0.383_sp   * REGR_4D(:,:,27,T)
-
-       ! Level 11:
-       Lct%Dct%Dta%V3(T)%Val(:,:,11) = 0.254_sp   * REGR_4D(:,:,27,T) &
-                                     + 0.746_sp   * REGR_4D(:,:,28,T)
-
-       ! Level 12:
-       Lct%Dct%Dta%V3(T)%Val(:,:,12) = 1.60e-2_sp * REGR_4D(:,:,28,T) &
-                                     + 0.984_sp   * REGR_4D(:,:,29,T)
-
-    ENDIF
-
-    ! Return with success
-    RC = HCO_SUCCESS
-
-  END SUBROUTINE GEOS5_TO_GEOS4_LOWLEV
-!EOC
-!------------------------------------------------------------------------------
-!                   Harmonized Emissions Component (HEMCO)                    !
-!------------------------------------------------------------------------------
-!BOP
-!
 ! !IROUTINE: COLLAPSE
 !
 ! !DESCRIPTION: Helper routine to collapse input levels onto the output grid.
 ! The input data is weighted by the grid box thicknesses defined on top of
 ! this module. The input parameter T determines the time slice to be considered,
-! and MET denotes the met field type of the input data (4 = GEOS-4 levels, GEOS-5
-! otherwise).
+! and MET denotes the met field type of the input data (22 = GISS, else GEOS-5).
```

Comment:
I know this was not introduced in this PR but "else GEOS-5" is potentially dangerous here. Maybe make `5 = GEOS-5` explicitly, keep `22 = GISS`, and throw error in the `ELSE`, so we don't have accidents calling this with an unsupported option (e.g. 4)

Reply:
Addressed.